testdata.log

```diff
@@ -8412,23 +8412,17 @@ SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 ) ORDER BY i;
 
 ---- 1349
 SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
```

Comment:
Hey, from reading about EXISTS.

It will only iterate on rows when the inner select evaluates to true. On the query above I assume the inner select evaluated to false hence no need to iterate at all.